libs/blocks/table/table.css

```diff
@@ -485,7 +485,6 @@
   text-align: start;
   font-style: normal;
   font-weight: 400;
-  gap: var(--spacing-xxs);
```

Comment:
Does this really need to be removed? It could mean that text isn't separated from the icon, right? Manually adding the space back doesn't seem to lead to any isses.

Reply:
<img width="305" alt="Screenshot 2024-10-30 at 9 03 15 AM" src="https://github.com/user-attachments/assets/1a8f1ed9-8574-4d71-b6b1-d8227356a427">

@overmyheadandbody I noticed that the icon actually gets styled to have margin on the left. So the gap was adding another extra space that was not in the design.